### PR TITLE
Prow configuration for pf-status-relay

### PR DESCRIPTION
**Pull request by mlguerrero12**
Run make new-repo for https://github.com/openshift/pf-status-relay

Manually removed e2e-aws job

### CODE REVIEW

**Comment by mlguerrero12** [comment]
rehearsals-ack

File: ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  base:
+    name: "4.17"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
```
Comment (SchSeba) [inline ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml:11]: the go version for 4.17(main) is 1.22

File: ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+promotion:
+  to:
+  - name: "4.17"
```
Comment (SchSeba) [inline ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml:14]: do we need also to publish the images to quay?

File: ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml (hunk 1 above)
Comment (mlguerrero12) [reply ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml:11]: The configuration script asks for the go version of the repo. It automatically adds it here. It should be backwards compatible, so the build should work. The pf-rehearse passed.

File: ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml (hunk 2 above)
Comment (mlguerrero12) [reply ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml:14]: yes, will do in a following pr.

File: ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml (hunk 1 above)
Comment (SchSeba) [reply ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml:11]: there is the all work done in openshift 4.17 to build with go toolchain 1.22.4 that is why I think for main (that points to 4.17) to have 1.22 if possible

File: ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml (hunk 2 above)
Comment (SchSeba) [reply ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml:14]: :+1:

File: ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml (hunk 1 above)
Comment (mlguerrero12) [reply ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml:11]: alright, will change it

Comment (mlguerrero12) [reply ci-operator/config/openshift/pf-status-relay/openshift-pf-status-relay-main.yaml:11]: @SchSeba, I set it back to 1.21 - CI was complaining that golang-1.22 does not exist. This is the initial configuration. I will use `rhel-9-golang-1.22-openshift-4.17` in the following pr when I set up publishing images to quay.

**Pull request merged**